server_api.py

```diff
@@ -457,9 +457,9 @@ def register_no_right_user(data):
 
         db = mysql()
         db.connect()
-        sql = "SELECT count(*) FROM user WHERE user_name='" + str(data["bluetooth_id"]) + "'"
-        pure_result = db.query(sql)
-        if int(pure_result[0][0])<1:
+        with UserDao() as userDao:
+            pure_result = userDao.checkUserExisted(userName=str(data["bluetooth_id"]))
+        if int(pure_result)<1:
```

Comment:
I think that above two lines can be more readable
* Change `pure_result` to `is_existed`
* `if int(pure_result) < 1:` -> `if not is_existed:`